libs/common/src/main/kotlin/person/Person.kt

```diff
@@ -22,7 +22,7 @@ data class Person(
     var kontaktadresse: List<Adresse>?,
     var oppholdsadresse: List<Adresse>?,
     val sivilstatus: Sivilstatus?,
-    val siviltilstand: List<Siviltilstand>?,
+    val siviltilstand: List<Siviltilstand>? = null,
```

Comment:
Hva er det denne brukes til nå som den aldri settes?

Reply:
Det brukes i GyldigOmstillingsSoeknadService for å vurdere om søknad er gyldig fremsatt for OMS. Den klassen skal fjernes da siden det skal skje manuelt av saksbehandler:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/889

Jeg tenkte å fjerne siviltilstand der samtidig som jeg fjerner GyldigOmstillingsSoeknadService